Assets/SuperMultiplayerShooter/Scripts/FriendsList/FriendsList.cs

```diff
@@ -223,6 +239,40 @@ public async Task<bool> PopulateFriendList(string channelGroup)
                     friendItem.Set(id, PNManager.pubnubInstance.CachedPlayers[id].Name);
                 }                  
             }
+
+            //Obtain history of messages from the friend request channel to determine if there are any pending invites.
+            PNResult<PNFetchHistoryResult> fetchHistoryResponse = await Connector.instance.GetPubNubObject().FetchHistory()
+                .Channels(new string[] { PubNubUtilities.chanFriendRequest + Connector.instance.GetPubNubObject().GetCurrentUserId() })
+                .ExecuteAsync();
+           if(fetchHistoryResponse != null && fetchHistoryResponse.Result != null && fetchHistoryResponse.Result.Messages != null && !fetchHistoryResponse.Status.Error)
```

Comment:
I thought by default the history API only returned something like 20 messages, but the documentation says 100.  I still think it is worth adding a comment that it's possible the API will only return SOME of the messages in the history (however unlikely that might be).  The documentation doesn't seem to say how you should know if there were additional messages which the API did not return, I thought there would be some kind of `more` flag but maybe I'm not understanding.